src/main/scala/com/hpe/sbt/package.scala

```diff
@@ -1,3 +1,19 @@
+/*
+ * Copyright 2018 Hewlett Packard Enterprise
```

Comment:
@eed3si9n @mkurz Since this is a new file I think that we may not need this copyright section? I can make adjustments to sbt-header so that it will detect existing source files that contain the HPE header and not report an error but create a clean Apache 2 header for new sources files.

Should I do this? We use this technique in Pekko as well

Reply:
Sounds good to me.

Reply:
Also according to the readme:

> This was originally written by ktoso for Akka, and extracted into a standalone sbt plugin and generalized by HPE.

so... can HPE claim the copyright? It might be best if we drop the whole copyright claim and just keep a small notice that the software is distributed under Apache-2.0? See for example what I do for Zinc - https://github.com/sbt/zinc/blob/develop/internal/zinc-core/src/main/scala/sbt/internal/inc/Incremental.scala

```scala
/*
 * Zinc - The incremental compiler for Scala.
 * Copyright Scala Center, Lightbend, and Mark Harrah
 *
 * Licensed under Apache License 2.0
 * SPDX-License-Identifier: Apache-2.0
 *
 * See the NOTICE file distributed with this work for
 * additional information regarding copyright ownership.
 */
```

Reply:
I have updated the project to use a header format as you suggested. I also disabled the update headers on compile and instead opted to add a header check in github actions since that will actually prevent the publish issue that just occurred (i.e. merging code with incorrect headers)

The `Headers / Check headers` should also be added as a branch check.